config/routes.rb

```diff
@@ -1,6 +1,10 @@
 FloodRiskEngine::Engine.routes.draw do
+  resources :enrollments
+
+  # Copied from dummy app - may be out of date now
   resources :enrollments, only: [:new] do
```

Comment:
didn't know what the nested stuff was supposed to be about so just did that to get going .. seemed to work fine but changed to above